### ISSUE TITLE
studio(team): social icons are not shown

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#814]
### Affected part of DeckDeckGo
- [x] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

### Reproduction

Steps to reproduce:
1. Go to https://deckdeckgo.com/team/
2. Look at social icons
 
 
### Expected Behavior

I expect to see five social icons.

<img width="548" alt="Screenshot 2020-07-28 at 13 56 21" src="https://user-images.githubusercontent.com/1763537/88662363-2460e800-d0da-11ea-9b0b-28da6499bd9c.png">
<br><br>
DeckDeckGo Team on localhost.

### Actual Behavior

I see three social icons.

<img width="528" alt="Screenshot 2020-07-28 at 13 58 08" src="https://user-images.githubusercontent.com/1763537/88662544-62f6a280-d0da-11ea-8de2-dc123ae5ee91.png">
<br><br>
DeckDeckGo Team on https://deckdeckgo.com/team/

<p>In the network tab, the icons **are** loaded, but when seeing the response, it's a replica of the webpage.</p>
<img width="400" alt="Screenshot 2020-07-28 at 14 00 47" src="https://user-images.githubusercontent.com/1763537/88662806-caaced80-d0da-11ea-8837-2f91bb6be253.png">

<p>The path for images might be wrong? It's hard to test local, because there everything looks fine.</p>

`<ion-icon src="./assets/icons/dev.svg" area-label="Dev"></ion-icon>`

### Environment

- Browser(s): Chrome, latest version. Also seen in Firefox, latest version.
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by peterpeterparker** [comment]
Not good:

```
<ion-icon src="./assets/icons/dev.svg" area-label="Dev"></ion-icon>
```

Good:

```
<ion-icon src="/assets/icons/dev.svg" area-label="Dev"></ion-icon>
```

Don't ask 🤷‍♂️

Thank you for the issue, should be solved with PR #818

**Comment by peterpeterparker** [comment]
P.S.: Could not test locally neither so I used our [staging](http://deckdeckgo-studio-beta.web.app/)

Funny thing, I also only noticed the error when accessing the page [team](http://deckdeckgo-studio-beta.web.app/team) directly. If I was navigating from home for example, all was fine.
